examples/hello-world-udp-proxy/supervisord.conf

```diff
@@ -0,0 +1,13 @@
+[supervisord]
+nodaemon=true
+
+[supervisorctl]
+
+[program:nginx]
+command=/usr/sbin/nginx -g 'daemon off;'
+
+[program:php-fpm]
+command=/usr/local/sbin/php-fpm -F
+
+[program:dvc-lb-proxy]
+command=/bin/bash -c "/usr/bin/devcycle-sdk-proxy -config /etc/devcycle-sdk-proxy/dvc-proxy-config.json && sudo chmod 777 /tmp/dvc_lb_proxy.sock"
```

Comment:
Might want to add a sleep in between - unless you tested this and it starts up fine

Reply:
I have not tested this update. Let me test it out real quick and determine if the sleep is necessary. Good catch!

Reply:
@JamieSinn Even with the sleep in between these commands, the file permissions aren't being set by the chmod command. I tried adding a `ENTRYPOINT` script with the command running after the supervisord command start, but that didn't work either. I'm still a bit green with creating Dockerfiles and docker-compose files, do you have any suggestions on what I might be doing wrong?

Reply:
Generally the tricky part is you're calling a single command shell, the easiest solution might be to create a script that handles the delay logic for you and then call that.

Alternatively, I could just merge the pr that's making it natively accessible in the proxy (the permissions)